bedrock/base/templates/404.html

```diff
@@ -2,47 +2,63 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% set_lang_files "mozorg/404" %}
-
-{% extends "base-pebbles.html" %}
+{% set_lang_files "mozorg/404-update-2020" %}
+{% extends "base-error.html" %}
 
 {% block gtm_page_id %}data-gtm-page-id="404"{% endblock %}
 
 {% block page_title %}{{ _('404: Page Not Found') }}{% endblock %}
 
-{# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
-{% block canonical_urls %}{% endblock %}
-
 {% block page_css %}
   {{ css_bundle('page_not_found') }}
 {% endblock %}
 
+{# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
+{% block canonical_urls %}{% endblock %}
+
 {% block content %}
-<main>
-  <div class="content">
-    <header>
-      <h1><span>{{ _('Whoops!') }}</span></h1>
-      <p>{{ _('Did you make a left at that last URL instead of a right? No problem. Here are some tips to get you back on your way:') }}</p>
-    </header>
-    <div class="error-content">
-      <ul class="prose">
-        <li>{{ _('If you typed in the address, check your spelling. Could just be a typo.') }}</li>
+  <main role="main">
+    <section class="mzp-l-content message-section">
+      <h2>{{ _("Sorry, we can't find that page") }}</h2>
+      <p>{{_("We're all about a healthy internet but sometimes broken URLs happen.") }}</p>
+      <div>
+        <a class="back-button" href="javascript:history.back()">{{ _("Go Back") }}</a>
+      </div>
+    </section>
+    <section class="mzp-l-content">
+      <ul>
         <li>
-        {% trans bugzilla='https://bugzilla.mozilla.org/enter_bug.cgi?product=www.mozilla.org' %}
-          If you’ve found an issue with one of our websites, we’d appreciate it if you could report the problem in Bugzilla, our <a href="{{ bugzilla }}">bug tracker</a>. One of our developers will take a look at it as soon as possible.
-        {% endtrans %}
+          <section class="mzp-has-icon">
+            <img class="list-icon" alt="" src="{{ static('protocol/img/icons/mozilla.svg') }}" width="30" height="30">
+              <p class="list-content">
+                {% trans about=url('mozorg.about') %}
+                  <a class="no-highlight" href="{{ about }}">Learn</a> about Mozilla, the non-profit behind Firefox.
```

Comment:
It might be worth trying to keep things like `class` names out of string translations, as it requires localizers to type everything correctly in order not to break styling. Instead you could try something like `.list-content a` in the CSS?